lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -134,7 +141,14 @@ static public function convertPositionalToNamedPlaceholders($statement)
      */
     public function bindValue($param, $value, $type = null)
     {
-        return $this->bindParam($param, $value, $type, null);
+        if ($value instanceof OCI8Descriptor) {
```

Comment:
So, this works, but Doctrine is still broken with the initial code.

Please try the `master` branch with the test case attached here, you'll get:

```
ORA-24816: Expanded non LONG bind data supplied after actual LONG or LOB column
```

Which I explained in previous comments. I'm all ears if you have a solution.